manifests/security/unattended_upgrades.pp

```diff
@@ -0,0 +1,38 @@
+# Make sure automatic security upgrades are activated
+# @param use_template  Create /etc/apt/apt.conf.d/50unattended-upgrades using a template
+class sunet::security::unattended_upgrades (
+  Boolean $use_template = false,
+) {
+  file { '/etc/dpkg/unattended-upgrades.debconf':
+    content => template('sunet/security/unattended-upgrades.debconf.erb'),
+  }
+  exec { 'enable_unattended_upgrades':
+    command => 'debconf-set-selections /etc/dpkg/unattended-upgrades.debconf && dpkg-reconfigure -fdebconf unattended-upgrades',
+    unless  => 'debconf-show unattended-upgrades | grep -q "unattended-upgrades/enable_auto_updates: true$"',
+    path    => ['/usr/local/sbin', '/usr/local/bin', '/usr/sbin', '/usr/bin', '/sbin', '/bin',],
+  }
+  if $use_template {
+    if $::facts['operatingsystem'] == 'Ubuntu' {
+      case $::facts['operatingsystemrelease'] {
+        '14.04':  { file { '/etc/apt/apt.conf.d/50unattended-upgrades':
```

Comment:
Minimun version to run the main branch is Ubuntu 22.04 so we need no legacy support here.

Reply:
Solved by discussing on Slack.